### PR TITLE
Update Dockerfile COPY failed

### PR DESCRIPTION
**Pull request by FDiskas**
FIX: 
```
ERROR: Service 'php' failed to build: COPY failed: stat /var/lib/docker/***/composer.lock: no such file or directory
```
 on first run `docker-compose up`

### CODE REVIEW

**Comment by FDiskas** [comment]
If I check container content I see that all correct files where copied
image

File: Dockerfile [hunk 1]
```diff
@@ -76,7 +76,7 @@ WORKDIR /srv/sylius
 ARG APP_ENV=prod
 
 # prevent the reinstallation of vendors at every changes in the source code
-COPY composer.json composer.lock symfony.lock ./
+COPY composer.json *.lock ./
```
[the remark this answers is not in the document]
Comment (FDiskas) [reply Dockerfile:79]: what about `yarn.lock` ?

[the remark this answers is not in the document]
Comment (tarnawski) [reply Dockerfile:79]: yarn.lock is copied a couple of lines below https://github.com/Sylius/Sylius-Standard/blob/017691dd49029a3ca5ae5e497d38ae82b0665422/Dockerfile#L124

**Comment by lchrusciel** [comment]
Hey @FDiskas,

thanks for your proposal and sorry for the lack of activity from our side. We are not docker specialists and it is hard to review these PRs. Can you check, if the problem, that you were trying to fix still exists after the merge of #536?

**Comment by lchrusciel** [comment]
Hey @FDiskas,

from what I know, this should be already fixed on master. Nonetheless, thanks a lot for your contribution.

**Pull request closed without merging**